sor/src/main/java/com/bazaarvoice/emodb/sor/core/DatabusEventWriter.java

```diff
@@ -0,0 +1,14 @@
+package com.bazaarvoice.emodb.sor.core;
+
+import java.util.Collection;
+
+/**
+ * Interface that should register with {@link DatabusEventWriterRegistry} and write events to the databus prior to
+ * writing them to the system of record. If an event fails to be written to the databus, then it should propagate and
+ * exception and prevent Emo from writing to the system of record.
+ */
+public interface DatabusEventWriter {
+
+
+    void writeEvent(Collection<UpdateRef> refs);
```

Comment:
This is very minor and I'm not going to push back on this, but I just wanted to note that the method name `writeEvent` made more sense when this method took a singular `UpdateIntentEvent`, but now the method takes a plural collection of objects so the cardinality seems wrong.  Maybe rename to something like `writeEvents`?